website/content/en/docs/faq.md

```diff
@@ -211,6 +211,8 @@ For information on upgrading Karpenter, see the [Upgrade Guide]({{< ref "./upgra
 
 ### How do I upgrade an EKS Cluster with Karpenter?
 
+Karpenter recommends that you always validate AMIs in your lower environments before using them in production environments. Read [Managing AMIs]({{<ref "./tasks/managing-amis" >}}) to understand best practices about upgrading your AMIs.
+
 When upgrading an Amazon EKS cluster, [Karpenter's Drift feature]({{<ref "./concepts/disruption#drift" >}}) can automatically upgrade the Karpenter-provisioned nodes to stay in-sync with the EKS control plane. Karpenter Drift is enabled by default starting `0.33.0`.
 
 {{% alert title="Note" color="primary" %}}
```

Comment:
I know this isn't really part of this PR, but since we're updating this FAQ do we want to drop the mention of defaulting to AL2 in the note? It really should just be that we default to the latest EKS optimized AMI for your selected family.

Reply:
Sure can make that change.